site_scons/components/__init__.py

```diff
@@ -22,11 +22,10 @@
 """Defines common components used by HPDD projects"""
 
 import sys
-import os
 import platform
 import distro
 from prereq_tools import GitRepoRetriever
-from prereq_tools import WebRetriever
+# from prereq_tools import WebRetriever
```

Comment:
Just remove?

Reply:
I thought about that on the basis that we no longer use it.  I should probably also remove the file, but there's a chance we'll want it again at some point.